src/main/java/com/gooddata/md/report/MetricGroup.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (C) 2007-2016, GoodData(R) Corporation. All rights reserved.
+ */
+
+package com.gooddata.md.report;
+
+/**
+ * Marker element marking the placement of metrics in Grid report.
+ * Can be contained either in rows or columns of {@link Grid}.
+ */
+public final class MetricGroup implements GridElement {
+
+    private final String value;
+
+    private static final String JSON_VALUE = "metricGroup";
+    private static final MetricGroup INSTANCE = new MetricGroup(JSON_VALUE);
```

Comment:
i would rather make it public and call it METRIC_GROUP (to use in static imports)